src/scss/packages/_usa-sidenav.scss

```diff
@@ -7,30 +7,22 @@
 
     &:hover {
       background-color: color('primary-lightest');
-      color: unset;
     }
   }
 
   .usa-current {
-    color: $theme-color-primary-dark;
+    color: color('primary-dark');
 
-    @include add-bar($theme-sidenav-current-border-width, 'secondary', 'left', 0, 0, 0);
-
-    @include at-media('tablet') {
-      @include add-bar($theme-sidenav-current-border-width, 'secondary', 'left', 0, 0, 0);
+    &::after {
+      background-color: color('secondary');
     }
 
     &:hover {
-      color: $theme-color-primary-dark;
+      color: color('primary-dark');
     }
   }
 }
 
-.usa-sidenav--sticky {
```

Comment:
was this unused or part of USWDS now?

Reply:
I'm glad you caught it, I had an internal debate about this, since it is unused across our projects, and doesn't exist in USWDS, but it's still technically a breaking change. I'm feeling a bit more comfortable with ripping out unused legacy cruft without bumping the major version, particularly for things which aren't documented anywhere. But I'm open to dissenting opinions!

Reply:
while there *could* be other consumers of this project besides us, AFAIK there aren't any. I think it's fine to break something like this, but it might be worth a release notes item

Reply:
I'll just restore it and then add it to a running set of breaking changes to be included in a future major release, like #298.

Reply:
I changed my mind. Let's keep the removal. I added a "Breaking Changes for Undocumented APIs" section to the CHANGELOG in 151c794, with a note about why it's not bumping the major version.